com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJoint.cs

```diff
@@ -138,17 +138,13 @@ private static UrdfJoint AddCorrectJointType(GameObject linkObject, JointTypes j
 #if  URDF_FORCE_ARTICULATION_BODY
 #else
 
-        
         /// <summary>
         /// Given a link, search up the tree to find a rigidbody.
         /// If a rigidbody is connected by a fixed joint, keep searching. 
         /// </summary>
-        /// <param name="linkObject"></param>
-        /// <returns></returns>
-        /// <exception cref="Exception"></exception>
-        public static Rigidbody FindCrucialParent(GameObject linkObject)
+        public static Rigidbody FindRigidBodyWithStaticJoining(Transform fromTransform)
         {
-            Transform currentTransform = linkObject.transform.parent;
```

Comment:
It is a strange choice to require that users know to provide the parent transform.

Reply:
The problem I had was previously the function always started at the parent. But then I needed a function that started at the transform, not the parent and didn't want to duplicate the function.